docker/README.md

```diff
@@ -0,0 +1,25 @@
+Installing Fineract with Docker and docker-compose
```

Comment:
can I suggest that we simply add this to the root README? It will be much easier for people to see, IMHO.